src/Backend/Modules/Analytics/Form/SettingsStepType.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+namespace Backend\Modules\Analytics\Form;
+
+use Backend\Core\Engine\TwigTemplate;
+use Common\ModulesSettings;
+use Google_Service_Analytics;
+
+/**
+ * An interface so we can split the settings form up into multiple forms
+ *
+ * @author Jelmer Prins <jelmer@sumocoders.be>
+ */
+interface SettingsStepType
```

Comment:
I do typehint to it https://github.com/forkcms/forkcms/pull/1534/files#diff-93d9fd5c110ab84c96cc9498a4391456R18